README.md

```diff
@@ -355,6 +355,8 @@ set AZURITE_ACCOUNTS="account1:key1:key2;account2:key1:key2"
 
 ### HTTPS Setup
 
+#### Mkcert
+
 You can generate a local certificate for 127.0.0.1 with `mkcert`. See the mkcert GitHub repo for [installation instructions](https://github.com/FiloSottile/mkcert), and then run the following commands.
```

Comment:
Please see my blog here for exact steps:  We don't want user to have to go off page to generate. https://blog.jongallant.com/2020/02/azurite-https-defaultazurecredential/